cmd/kubermatic-api/swagger.json

```diff
@@ -21818,7 +21844,7 @@
     },
     "AKSLocation": {
       "type": "object",
-      "title": "AKSLocation is the object representing Azure Location.",
+      "title": "AKSLocation represents an object representing Azure Location.",
```

Comment:
`AKSLocation represents an object representing Azure Location.` Please re-word this